freqtrade/rpc/rpc.py

```diff
@@ -135,12 +134,11 @@ def _rpc_status_table(self) -> DataFrame:
                     trade.pair,
                     shorten_date(arrow.get(trade.open_date).humanize(only_distance=True)),
                     f'{trade_perc:.2f}%'
+
```

Comment:
remove it? (empty line)

Reply:
it kindof is? (not sure - i think you're looking at a previous commit?)